org.openhab.binding.zigbee.cc2531/src/main/java/org/openhab/binding/zigbee/cc2531/internal/CC2531HandlerFactory.java

```diff
@@ -47,6 +49,18 @@
     private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections
             .singleton(CC2531BindingConstants.THING_TYPE_CC2531);
 
+    @Nullable
+    private ZigBeeChannelConverterFactory zigbeeChannelConverterFactory;
+
+    @Reference
```

Comment:
Adding the `@Reference` directly on the field would remove the need for the `set` and `unset` methods.